morpheus/_lib/src/stages/filter_detection.cpp

```diff
@@ -27,18 +27,17 @@
 namespace morpheus {
 // Component public implementations
 // ************ FilterDetectionStage **************************** //
-FilterDetectionsStage::FilterDetectionsStage(const neo::Segment &parent, const std::string &name, float threshold) :
-  neo::SegmentObject(parent, name),
-  PythonNode(parent, name, build_operator()),
+FilterDetectionsStage::FilterDetectionsStage(float threshold) :
+  PythonNode(base_t::op_factory_from_sub_fn(build_operator())),
   m_threshold(threshold)
 {}
 
-FilterDetectionsStage::operator_fn_t FilterDetectionsStage::build_operator()
```

Comment:
Greatly appreciate the name change, improves intuition/readability.

Reply:
Great